aws-analytics-pinpoint/src/main/java/com/amplifyframework/analytics/pinpoint/AmazonPinpointAnalyticsPlugin.java

```diff
@@ -324,14 +324,6 @@ public void configure(@NonNull JSONObject pluginConfiguration, @NonNull Context
                                 .getLong(PinpointConfigurationKey.AUTO_FLUSH_INTERVAL.getConfigurationKey()));
             }
 
-            if (pluginConfiguration
```

Comment:
How come this is getting removed? Isn't this PR about changing visibility of class/feild members? But, this looks like a functional change.

Reply:
Oh I should have kept it separate but this does not change the functionality. This flag is not required since the session is defined by app being in foreground and background rather than by time interval. So concept of time interval doesn't make sense anymore.